octopose.py

```diff
@@ -27,21 +27,21 @@
 import argparse
 import shutil
 import os
-import subprocess
 import sys
 import octo
 import nu
+import util
 
 
-def invoke_deploy(step_path):
+def invoke_deploy(step_path, verbose):
```

Comment:
Is there a way to do this without having to pass `verbose` to every method? and jsut have `run_subprocess` know from the start if it should be verbose or not?

Reply:
Unsure if I've gone a little too much towards classes with the refactoring, but I think there's now a slightly nicer separation of code. Interested to hear your thoughts.